molecule/pxc/pxc-innovation-lts-bootstrap-upgrade/tasks/main.yml

```diff
@@ -0,0 +1,150 @@
+    when: lookup('env', 'install_repo') == "testing" 
+
+  - name: include tasks for enabling main repo pxc 80
+    include_tasks: ../../../../tasks/enable_pxc80_main_repo.yml
+    when: lookup('env', 'install_repo') == "main" or lookup('env', 'install_repo') == ""
+
+  - name: include tasks for enabling experimental repo pxc 80
+    include_tasks: ../../../../tasks/enable_pxc80_experimental_repo.yml
+    when: lookup('env', 'install_repo') == "experimental"
+
+  - name: install python3-libselinux 
+    yum:
+      name: "{{ packages }}"
+      state: present
+    vars:
+      packages:
+      - python3-libselinux
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "8"
+
+  - name: allow all users to connect to mysql
+    shell: setsebool -P mysql_connect_any 1
+    when: ((ansible_os_family == "RedHat") and (ansible_distribution != "Amazon"))
+
+  - name: flush iptables
+    iptables:
+      flush: yes
+    when: ansible_os_family == "RedHat"
+
+  - name: install PXC 8 deb packages
+    apt:
+      name: "{{ packages }}"
+      update_cache: yes
+    vars:
+      packages:
+      - percona-xtradb-cluster-full
+      - percona-toolkit
+    when: ansible_os_family == "Debian"
+    environment:
+      PERCONA_TELEMETRY_URL: "https://check-dev.percona.com/v1/telemetry/GenericReport"
+      PERCONA_TELEMETRY_DISABLE: "1"
+
+  - name: install PXC {{ major_release_version}} rpm packages
+    yum:
+      name: "{{ packages }}"
+      state: latest
+    vars:
+      packages:
+      - percona-xtradb-cluster-full
+      - percona-toolkit
+    when: ansible_os_family == "RedHat"
+    environment:
+      PERCONA_TELEMETRY_URL: "https://check-dev.percona.com/v1/telemetry/GenericReport"
+      PERCONA_TELEMETRY_DISABLE: "1"
+
+  - name: stop mysql service with service command
+    command: systemctl stop mysql
+
+  - name: copy pxc config on centos
+    template: src=../../../../templates/my_rpm_80.j2 dest=/etc/my.cnf
+    when: ansible_os_family == "RedHat"
+
+  - name: copy pxc certs
+    copy:
+      src: ../../../../support-files/certs/
+      dest: /etc/mysql/certs/
+
+  - name: configure PXC on debian/ubuntu
+    template: src=../../../../templates/my_8.j2
+              dest=/etc/mysql/my.cnf
+              mode=0640 owner=mysql group=root
+    when: ansible_os_family == "Debian"
+
+  # bootstrap the cluster
+  - name: bootstrap cluster on CentOS
+    command: systemctl start mysql@bootstrap.service
+
+  # set root password on centos
+  - name: set root password on centos
+    command: /package-testing/setpass_57.sh
+    when: ansible_os_family == "RedHat"
+
+  - name: copy .my.cnf with credentials on centos
+    template: src=../../../../templates/my_57.j2
+              dest=/root/.my.cnf
+              mode=0640 owner=root group=root
+    when: ansible_os_family == "RedHat"
+
+  # other tests
+# - name: install plugins
+#   command: /package-testing/plugins_test_80.sh pxc
+
+  - name: register PXC version Molecule
```

Comment:
Why do we need this if we have it in VERSIONS file?

Reply:
This step is for checking the installed pxc version 
after this task we have another task where we print value using ansible debug module